allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -18,8 +18,14 @@
 LONG_STIM_THRESHOLD = 0.2     # seconds
 MAX_MONITOR_DELAY = 0.07     # seconds
 
+"""
+These are the old legacy key pairs, I kept these in comments because
+I want a record of what we assumed the key pairs looked like at one
```

Comment:
commit history keeps a record for us. Clean code is readable code. This is extra.

Reply:
what action was taken to resolve this conversation?

Reply:
Removed comment